src/compat/posix/include/utmp.h

```diff
@@ -60,7 +60,7 @@ struct utmp {
 	int32_t ut_addr_v6[4];        /* Internet address of remote
 					host; IPv4 address uses
 					just ut_addr_v6[0] */
-	char __unused[20];            /* Reserved for future use */
+	//char __unused[20];            /* Reserved for future use */
```

Comment:
Why did you remove this line? Is it related to the quque.h somehow?

Reply:
> Please don't create a new pull request for every fix, just push more commits to the existing branch.

Ok, sir. I'll do that

Reply:
> Why did you remove this line? Is it related to the quque.h somehow?

It has been already defined in cdefs.h . So I removed it.